app/models/customer.rb

```diff
@@ -6,8 +6,17 @@ class Customer < ActiveRecord::Base
   has_many :orders, class_name: Spree::Order
   before_destroy :check_for_orders
 
+  belongs_to :bill_address, foreign_key: :bill_address_id, class_name: Spree::Address
+  alias_attribute :billing_address, :bill_address
```

Comment:
It's like a kind of convention from Spree, I learned it from Spree::Order and Spree::User, it's not used much.